src/utils/IdealGraphVisualizer/Bytecodes/pom.xml

```diff
@@ -30,19 +30,21 @@
  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 -->
 
-<project xmlns="https://maven.apache.org/POM/4.0.0" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
It rolled back changes from https://bugs.openjdk.org/browse/JDK-8277869
IDEA works fine with `https`. I think we should make it `https` again